docs/static/jvm.asciidoc

```diff
@@ -5,7 +5,7 @@
 {ls} requires one of these versions:
 
 * Java 11
-* Java 17 (see <<jdk17-upgrade>> for settings info)
+* Java 17 (default, see <<jdk17-upgrade>> for settings info)
```

Comment:
```suggestion
* Java 17 (default) Check out <<jdk17-upgrade>> for settings info.
```

Reply:
This construct looks cleaner, and uses language that's more inclusive ("Check out" instead of "see.")